sound/tts/sound.txt

```diff
@@ -886,7 +886,7 @@
     ro = Apoi
     ja = その先
     da = Derefter
-    es = Después
+    es = Después.
     fi = Sitten.
```

Comment:
@bykoianko ok but do you want me to modify the "fi" language too? This PR was made for the Spanish translation.